cancunDevs/omiqroo-front
# Add React Router 4.2.2

Add React Router v4.2.2 and create the simple pages for the main rout
es.

Main routes:
	- Inicio
	- Convocatoria
	- Material
	- Anuncios
	- Contacto
	- Resultados

Install npm package react-router-dom, add the BrowserRouter component
at the outest level of the app. Create links with the NavLink
component and use Switch with Route components to render the correct
components to the page.

#### What does this PR do?

Adds routing to the app by the use of React Router 4.2.2 

#### Where should the reviewer start?

On file `src/index.js`

#### How should this be manually tested?

1. Run `npm start`
2. On the browser click on the different links
3. Check the page rendered after the click it's the one that matches the link

#### Any background context you want to provide?

The different pages for the main sections are just for a demostrative purpose.

#### What are the relevant tickets?

Closes [#2 Add React Router](https://github.com/cancunDevs/omiqroo-front/issues/2)
